### PR TITLE
Home Screen UI Update

### PR DESCRIPTION
**Pull request by ryankittle98**
New Things Added :

- Added rounded_button.xml to make buttons with rounded corners
- Added a golden version of the ThinQ.tv brain icon
- Added a video file and related code to use as homescreen background, but then commented it out. The video kinda clutters the screen, and I'm not sure if a basic home screen would look better. Will ask your guys' input next time I see you.
- I also commented out the original text field/button combo, because the Join button should only link to the default ThinQ.tv chatroom as it does on the website

Actions Accomplished :
- I made the home screen look prettier. It's now got a header, two buttons to Join/Get Involved, and a ScrollView for Upcoming Conversations.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -31,4 +31,5 @@ dependencies {
     androidTestImplementation 'androidx.test.espresso:espresso-core:3.1.1'
 
     implementation ('org.jitsi.react:jitsi-meet-sdk:2.+') {transitive = true}
+    implementation 'androidx.recyclerview:recyclerview:1.1.0'
```
Comment (taangle) [inline app/build.gradle]: Is this a necessary addition?

File: app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java [hunk 2]
```diff
@@ -32,10 +37,19 @@ protected void onCreate(Bundle savedInstanceState) {
                 .setWelcomePageEnabled(false)
                 .build();
         JitsiMeet.setDefaultConferenceOptions(defaultOptions);
+
+        // COMMENTED OUT BECAUSE I'M NOT SURE IF IT ACTUALLY LOOKS GOOD. GET TEAM OPINION
+        //mainViewer = (VideoView) findViewById(R.id.videoview);
+        //Uri uri = Uri.parse("android.resource://" + getPackageName() + "/" + R.raw.crowdpubcommercialv_1);
+        //mainViewer.setVideoURI(uri);
+        //mainViewer.start();
```
Comment (taangle) [inline app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java]: We probably shouldn't commit commented code like this to master. Instead make a branch where this code is uncommented and anyone can take a look at it, then remove it entirely here.

Comment (taangle) [reply app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java]: Also, when removing code, make sure you remove any unused imports.

File: app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java [hunk 3]
```diff
@@ -32,10 +37,19 @@ protected void onCreate(Bundle savedInstanceState) {
...
     }
 
+// COMMENTED OUT BECAUSE BUTTON IS NOT USEFUL ANYMORE
```
Comment (taangle) [inline app/src/main/java/com/thinqtv/thinqtv_android/MainActivity.java]: Same here - just remove the code entirely, we have access to it in other branches and past commits.

File: app/src/main/res/layout/activity_main.xml [hunk 4]
```diff
@@ -4,37 +4,152 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
+    android:background="@color/ic_launcher_background"
     tools:context=".MainActivity">
 
+<!--  COMMENTED OUT BECAUSE BUTTONS ARE NO LONGER USEFUL /AND/ NOT SURE IF VIDEO IS GOOD IDEA
```
Comment (taangle) [inline app/src/main/res/layout/activity_main.xml]: And again here. I mean, it's debatable whether commiting commented code is a good practice or not, but I don't think Lisa would like it.

**Review by taangle — CHANGES_REQUESTED** [review]
Just a few changes, mostly getting rid of commented code.

File: app/src/main/res/layout/activity_main.xml (hunk 4 above)
Comment (ece106) [reply app/src/main/res/layout/activity_main.xml]: Yeah, garbage in my code is not good.

**Pull request merged**